src/main/java/seedu/address/logic/commands/MarkAttendanceCommand.java

```diff
@@ -63,6 +64,7 @@ public CommandResult execute(Model model) throws CommandException {
 
         // Mark attendance
         student.markAttendance(date, attendance.value);
-        return new CommandResult(String.format(MESSAGE_SUCCESS, name, attendance, date));
+        return new CommandResult(String.format(MESSAGE_SUCCESS, name, attendance,
+                DateTimeFormatter.ofPattern("MMM d yyyy").format(date)));
```

Comment:
Do we want to update the formatting for the date in GetAttendanceCommand as well?